File: algorithm/algorithm.py
def hello_wold(n):
    for i in range(n):
        if i % 3 == 0:
            print("Hello")
        elif i % 5 == 0:
            print("World")
        elif i % 7 == 0:
            print("Yoo")
        else:
            print(i)    

hello_wold(100)

File: algorithm/algorithm.js
function helloWorld(n) {
  for (let i = 1; i <= n; i++) {
    if (i % 3 === 0) {
      console.log("Hello");
    } else if (i % 5 === 0) {
      console.log("World");
    } else if (i % 7 === 0) {
      console.log("Yoo");
    } else {
      console.log(i);
    }
  }
}

helloWorld(100);
